doc/LocalVMSetup.md

```diff
@@ -16,9 +16,16 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
 ## 3. Install a fresh CentOS 7.8.2003
 Automatic install or manual install. Basic installation is OK. When you build CORTX from source in next steps, dependant packages will be resolved and installed.
 
-## 4. You may need to add EPEL repo.
+## 4. Install GUI so you can setup ssh key
+yum groups install "GNOME Desktop"
+
+startx
```

Comment:
Why is this needed?

Reply:
I think this is helpful because I spent a lot of time trying to figure out how to get passed the ssh step in the quick start guide and by having a GUI it makes the ssh step a lot more straight forward

Reply:
GNOME Desktop or X11 graphics definitely should not be needed to pass the "ssh step".

Note: the VM machine is designed to work as a server node, not a desktop one.

Reply:
Ok, but the way I figured out to do it was to use the Desktop GUI, how would you do it in command line?

Reply:
Then, what is the problems in the "ssh step"?
What specific things that you have troubles to do in command line?

Reply:
The problem I was having was that I wasn't able to get access, I was supposed to put in a key to my GitHub account

Reply:
You can do this in your host OS, not in VM.

Reply:
Ok, but do you hve to log into you github account then from the command line?